### PR TITLE
C-1606 C-1609 C-1610 Share to IG Story QA

### PR DESCRIPTION
**Pull request by nicoback2**
### Description
-Use showfreqs instead of showwaves for a different look (can revert)
-Fix Android not working
-Fix gradient being flaky
-Make loading faster

https://user-images.githubusercontent.com/36916764/206299391-9300cfc8-ed3e-4e6d-8a3f-568035aada20.mov

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/mobile/android/app/build.gradle [hunk 1]
```diff
@@ -144,6 +144,13 @@ android {
     compileSdkVersion rootProject.ext.compileSdkVersion
     flavorDimensions "default"
 
+    packagingOptions {
```
Comment (dylanjeffers) [inline packages/mobile/android/app/build.gradle:147]: wild, whats the tldr here?

File: packages/mobile/android/build.gradle [hunk 2]
```diff
@@ -8,7 +8,7 @@ buildscript {
         targetSdkVersion = 31
         castFrameworkVersion = "21.0.0"
         ndkVersion = "21.4.7075529"
-        ffmpegKitPackage = "https-lts"
+        ffmpegKitPackage = "https-gpl-lts"
```
Comment (dylanjeffers) [inline packages/mobile/android/build.gradle:11]: i guess this too? need GPL version for android?

File: packages/mobile/src/components/share-drawer/useShareToStory.tsx [hunk 3]
```diff
@@ -274,7 +273,7 @@ export const useShareToStory = ({
 
       // Step 4: Put everything together and push to IG
       const shareOptions = {
-        backgroundVideo: storyVideoPath,
+        backgroundVideo: `file://${storyVideoPath}`,
```
Comment (dylanjeffers) [inline packages/mobile/src/components/share-drawer/useShareToStory.tsx:276]: nice

**Review by dylanjeffers — APPROVED** [review]
nice fixes, cant speak to the showwaves changes, but lgtm!

**Review by sliptype — APPROVED** [review]
Nice work, gradient is looking awesome! Let's get forrest/julian's input on the new waves. I wonder how they would look if they were semi transparent?

File: packages/mobile/android/build.gradle (hunk 2 above)
Comment (nicoback2) [reply packages/mobile/android/build.gradle:11]: Yeah so we needed to use the libx264 encoder to produce the properly formatted video with H.264/AAC, which is included in the GPL version but not the regular one. then i also needed to add `file://` (for some reason it was working without this in iOS, it's like a small print thing in the react-native-share docs)

Comment (dylanjeffers) [reply packages/mobile/android/build.gradle:11]: Awesome finds, wow

File: packages/mobile/android/app/build.gradle (hunk 1 above)
Comment (nicoback2) [reply packages/mobile/android/app/build.gradle:147]: Oh this was just to fix [this issue](https://github.com/arthenica/ffmpeg-kit/wiki/Tips#2-depending-another-android-library-containing-libc_sharedso) where two of our packages included `libc++_shared.so`

**Pull request merged**